scala/scala.bzl

```diff
@@ -700,3 +704,5 @@ def scala_specs2_junit_test(name, **kwargs):
         suite_class = "io.bazel.rulesscala.specs2.Specs2DiscoveredTestSuite",
         **kwargs
     )
+
+scala_doc = _scala_doc
```

Comment:
I'm following a pattern I see in [other rules](https://github.com/tweag/rules_haskell/blob/1375cc4ac8fb30474113a62f0daa9200dc42e08a/haskell/haskell.bzl#L354-L386), in which you basically re-export rules that are loaded from other .bzl files, makes it such that a user can load multiple rules from the same .bzl file.